bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -55,7 +56,10 @@ android {
 	}
 
 	externalNativeBuild {
-		@CMAKE_PATH@
+		cmake {
+			// version "3.18.1"
```

Comment:
We have Android requirements listed at https://github.com/KhronosGroup/Vulkan-Samples/wiki/Known-Issues

But with all the stuff getting reworked it may be a better idea to have this in the BUILD.md instead.

Reply:
This Pull Request depends on https://github.com/KhronosGroup/Vulkan-Samples/pull/565: I would like to recommend to do explicit configuration for NDK and CMake versions, people would not need to worry about them anymore.  Once #565 is in, those versions are always set.

Reply:
for the pointed out wiki page, let me think about it and update it accordingly. I think if we do the following:
 - explicitly set the versions in our app/build.gradle: so they got installed automatically, or got told to install the required versions if license has not been accepted on that machine.
 - avoid asking people for local Environment variables for Android gradle purpose (no need). If the same tools(NDK/CMAKE for example) are also needed for other purpose on the system, set environment variables to point to the SDK installed versions.
 
with the above, we might help developer avoiding confusions. I will update the wiki page and have you review then, can be separate from this Pull Request.